### PR TITLE
merge pkg.template into pkg.mysqlclient

### PR DESCRIPTION
**Pull request by caseymcc**
This is a pkg.template merge, not sure how they are handled in general. This includes the merge and the changes for travis and appveyor. Let me know if this needs to be handled a different way.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -4,56 +4,44 @@ environment:
...
-     - TOOLCHAIN: "nmake-vs-12-2013"
-       PROJECT_DIR: examples\MySQL-client
+#    https://ci.appveyor.com/project/caseymcc/hunter/builds/19950045/job/kf5bbug53mwuv52y
+#    - TOOLCHAIN: "vs-15-2017-win64-cxx17"
+#      PROJECT_DIR: examples\MySQL-client
+#      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
-     - TOOLCHAIN: "vs-10-2010"
-       PROJECT_DIR: examples\MySQL-client
+    - TOOLCHAIN: "vs-14-2015-win64"
```
Comment (ruslo) [inline appveyor.yml:46]: It's `vs-14-2015-sdk-8-1`, not `vs-14-2015-win64`:
* https://github.com/ingenue/hunter/blob/3650bcbc792940034cdbad7f7f05051dd5f1f68c/appveyor.yml#L23-L25

If you want to change toolchain then it's better to comment out default one and add another to the bottom.

**Review by ruslo — CHANGES_REQUESTED** [review]
> Conflicting files
> .travis.yml
> appveyor.yml

Can't do the merge because of conflicts.

**Comment by caseymcc** [comment]
I don't understand why there is a merge problem to begin with. I resolved the merge problem when I merged the pkg.template branch then updated the files with another commit. Shouldn' t the merge already be resolved? Why do I have to resolve it again on github?

**Comment by caseymcc** [comment]
Needs updated mysql https://github.com/ruslo/hunter/pull/1605, in order for CI to pass.

**Comment by ruslo** [comment]
> I don't understand why there is a merge problem to begin with. I resolved the merge problem when I merged the pkg.template branch then updated the files with another commit. Shouldn' t the merge already be resolved? Why do I have to resolve it again on github?

Sounds like a right path, hard to say what went wrong without details.

> Needs updated mysql ruslo#1605, in order for CI to pass.

If your patch fix some issues you should test it against old toolchains first, then add new toolchains after merge, see:
* https://docs.hunter.sh/en/latest/creating-new/update.html#add-toolchains

**Comment by caseymcc** [comment]
Updated the test to include the originals before the merge

https://travis-ci.org/caseymcc/hunter/builds/452039939
https://ci.appveyor.com/project/caseymcc/hunter/builds/20126897

**Review by ruslo — CHANGES_REQUESTED** [review]
> Conflicting files
> .travis.yml
> appveyor.yml

I can't merge

**Comment by caseymcc** [comment]
fixed the merge, still not sure why it is happening.

File: .travis.yml [hunk 2]
```diff
@@ -8,107 +8,177 @@ language:
...
-    - os: linux
-      env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=gcc
 
-    #  FIXME:
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505605
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
+    - os: linux
+      env: > 
+        TOOLCHAIN=default
```
Comment (ruslo) [inline .travis.yml:38]: There is no `default` toolchain in testing table:
* https://github.com/ingenue/hunter/blob/6cf263ab8d7ed931de20ef210a353940ad4c6b36/.travis.yml#L36-L69

It was removed a long time ago...

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by ruslo** [comment]
I have updated branch, testing:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20129988
* https://travis-ci.org/ingenue/hunter/builds/452093991?utm_source=github_status&utm_medium=notification

**Pull request closed without merging**